app/pages/organizations/organization-container.jsx

```diff
@@ -60,13 +66,31 @@ class OrganizationContainer extends React.Component {
     this.fetchProjects(this.state.organization, newView);
   }
 
-  fetchProjects(organization, collaboratorView) {
-    this.setState({ errorFetchingProjects: null, fetchingProjects: true });
+  updateQuery(newParams) {
+    const query = Object.assign({}, this.props.location.query, newParams);
+    const results = [];
+    Object.keys(query).map((key) => {
+      if (query[key] === '') {
+        results.push(delete query[key]);
+      }
+      return results;
```

Comment:
Does the results array get used somewhere again? If not, maybe `forEach` could be used instead of `map`?